brozzler/js-templates/try-login.js.j2

```diff
@@ -1,6 +1,11 @@
 var __brzl_tryLoginState = 'trying';
 
 var __brzl_tryLogin = function() {
+    for(var x = 0; x < document.querySelectorAll("[class^='login-open']").length; x++){
+        button = document.querySelectorAll("[class^='login-open']")[x];
```

Comment:
You are running `querySelectorAll` too many times here, N times in the `for` loop and once more to get the `button`.
You could do `var login_open = document.querySelectorAll("[class^='login-open']")` and use that.